templates/case/_case_show/_resident_and_rent_board_shareds.html.twig

```diff
@@ -154,7 +154,14 @@
                         </div>
                         <div class="form-group col-md-3">
                             <label>{% trans %}Regulated rent{% endtrans %}</label>
-                            <input type="text" class="form-control" value="{{ case.leaseRegulatedRent }}" readonly>
+                            <input type="text" class="form-control" value="
+                                {% if case.leaseRegulatedRent is not null %}
+                                    {% if case.leaseRegulatedRent %}
+                                        {% trans %}Yes{% endtrans %}
+                                    {% else %}
+                                        {% trans %}No{% endtrans %}
+                                    {% endif %}
```

Comment:
Can be written as `{{ case.leaseRegulatedRent ? 'Yes'|trans : 'No'|trans }}`.

Reply:
The idea was to not display anything if `case.leaseRegulatedRent` was not set (i.e. `null`) and `Yes`/ `No` if either was explicitly set. But using ternary operator does still make sense.

Switched to
`{{ case.leaseRegulatedRent is not null ? (case.leaseRegulatedRent ? 'Yes'|trans : 'No'|trans )}}`